realm/realm-library/src/androidTest/java/io/realm/RealmTests.java

```diff
@@ -1922,14 +1911,15 @@ public void run() {
         realm.beginTransaction();
         realm.createObject(AllTypes.class);
         realm.commitTransaction();
+        // Move the directory so core won't be able to delete log files
```

Comment:
Instead describe why it could happen, i.e during an upgrade of Realm where core changed the location of log files.